include/tins/sniffer.h

```diff
@@ -578,8 +578,13 @@ namespace Tins {
         for(iterator it = begin(); it != end(); ++it) {
             try {
                 // If the functor returns false, we're done
+                #if TINS_IS_CXX11
+                if (!Tins::Internals::invoke_loop_cb<Functor, Packet>(function, *it))
```

Comment:
I don't see a reason to invoke the function using explicit template parameters. Is there some case in which template parameter deduction doesn't work as expected here?